packages/suite/src/views/wallet/sign-verify/components/SignMessage/index.tsx

```diff
@@ -0,0 +1,249 @@
+    });
+    if (
+        account.networkType === 'bitcoin' &&
+        typeof account.addresses?.unused[0]?.address === 'string' &&
+        typeof account.addresses?.unused[0]?.path === 'string'
+    ) {
+        const accountsSelectArr = [...account.addresses.unused, ...account.addresses.used];
+        accountsSelectOpt = accountsSelectArr.map(el => ({
+            label: el.address,
+            value: el.path,
+        }));
+    } else if (account.networkType === 'ethereum') {
+        address = account.descriptor;
+        path = account.path;
+    }
+
+    const { getValues, setValue, errors, formState, reset, control, register, watch } = useForm<
+        Inputs
+    >({
+        defaultValues: {
+            address,
+            path,
+            signMessage: '',
+            accountsSelect: { label: '', value: '', type: '' },
+            signature: '',
+        },
+        mode: 'onChange',
+    });
+    const { isValid } = formState;
+    const watchSignature = watch('signature');
+
+    useEffect(() => {
+        return () => {
+            reset(); // clear data after coin change
+            path = '';
+            address = '';
+        };
+    }, [reset]);
+
+    return (
+        <Card title={<Translation id="TR_SIGN_MESSAGE" />} isColumn>
+            <Column>
+                <Row>
+                    {account.networkType === 'bitcoin' && (
+                        <Controller
+                            as={<StyledSelect options={accountsSelectOpt} />}
+                            control={control}
+                            topLabel={<Translation id="TR_ADDRESS" />}
+                            name="accountsSelect"
+                            state={errors.accountsSelect ? 'error' : undefined}
+                            onChange={([selected]) => {
+                                path = selected.value;
+                                address = selected.label;
+                                return selected;
+                            }}
+                            bottomText={
+                                errors.accountsSelect && (
+                                    <Translation
+                                        id={(errors.accountsSelect.type as any) as errorsMessage}
+                                    />
+                                )
+                            }
+                            rules={{
+                                validate: {
+                                    TR_REQUIRED_FIELD: (selectValue: Inputs['accountsSelect']) => {
+                                        return selectValue.value.length > 0;
+                                    },
+                                },
+                            }}
+                        />
+                    )}
+                    {/* input is more convenient from the ability to select and copy */}
+                    {account.networkType === 'ethereum' && (
+                        <Input
```

Comment:
use disabled `Select` with one option. You shouldn't be able to copy address unverified  by device

Reply:
input was left there because of the old feature, where you can copy the address, ok i rewrite